agent/ansible/collection/roles/pbench_firewall_open_ports/README.md

```diff
@@ -14,7 +14,7 @@ These are the variables that are defined by default and do not
 generally need to be modified unless there is a conflict:
 
 - pbench_redis_port: 17001
-- pbench_wsgi_port: 8080
```

Comment:
This is kinda interesting since (A) one can in theory happy use Pbench Agent without ever caring about the Pbench Server and now (B) we might only talk to a Relay with an arbitrary port instead of the server. This doesn't make anything worse, so c'est la vie, and I've got no idea what if anything we do about it going forward. Technically all this stuff is "examples and building blocks" anyway.

Reply:
Oh. Yeah, you're probably right at that.